src/winston.js

```diff
@@ -4,27 +4,34 @@
  */
 
 require('le_node');
-var winston = require('winston');
+const winston = require('winston');
 
-module.exports = function(token) {
-  //we do not want console transport in production, because it reduce performances
+module.exports = (notifier, token, notify = true) => {
```

Comment:
Are we sure we want to notify to be true? This means that projects once updated will send all warns and errors to bugsnag (hopefully not tests also). I mean it makes sense, just checking if you have talked about this with @PuKoren about the impact?